crates/modelardb_server/src/parser.rs

```diff
@@ -86,6 +86,16 @@ impl ModelarDbDialect {
         parser.expect_keyword(Keyword::TABLE)?;
         let table_name = self.parse_word_value(parser)?;
 
+        // Check that the table name is not a restricted keyword.
+        for keyword in ALL_KEYWORDS {
+            if table_name == keyword.to_uppercase() || table_name == keyword.to_lowercase() {
+                return Err(ParserError::ParserError(format!(
+                    "Reserved keyword '{}' cannot be used as a table name.",
+                    table_name.to_uppercase()
```

Comment:
Maybe remove `to_uppercase()` so the user sees the exact data they entered into the system instead of the data we have transformed and thus, hopefully, easier can understand what the problem is.

Reply:
Fixed.